### PR TITLE
update elixir version (to fix benchmark)

### PR DESCRIPTION
**Pull request by thousandsofthem**
Latest preview run for Phoenix is failing due to outdated elixir version. 
This pull request updates elixir version to prevent issue with compiling phoenix

http://tfb-logs.techempower.com/round-13/preview-1/latest/logs/phoenix/

### CODE REVIEW

File: toolset/setup/linux/languages/elixir.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ RETCODE=$(fw_exists ${IROOT}/elixir.installed)
   return 0; }
 
 ELIXIR_HOME=$IROOT/elixir
-VERSION="1.1.0-1"
+VERSION="1.2.6-1"
```
Comment (stevedomin) [inline toolset/setup/linux/languages/elixir.sh]: Why not use 1.3.2?

**Comment by thousandsofthem** [comment]
possible, just not sure it'll work flawlessly with the old code. should i update to 1.3?

**Comment by stevedomin** [comment]
I think we should if possible, I started working on upgrading Phoenix as well over the w-e, will try to wrap up tonight

**Comment by ckampfe** [comment]
@thousandsofthem @stevedomin I did a bit of this work already https://github.com/TechEmpower/FrameworkBenchmarks/pull/2214
Feel free to borrow what you need :)

**Comment by thousandsofthem** [comment]
Mine is quick patch to avoid missing yet another round (so sad)
yours is vastly superior - great work!

**Comment by nbrady-techempower** [comment]
I'm going to close this in favor of #2214. Thank you @thousandsofthem It looks like we are going to get that into round-13.

**Comment by thousandsofthem** [comment]
Great, thanks!

**Pull request closed without merging**